target/linux/qualcommax/image/ipq807x.mk

```diff
@@ -332,6 +332,21 @@ define Device/redmi_ax6
 endef
 TARGET_DEVICES += redmi_ax6
 
+define Device/tplink_deco-x80-5g
+	$(call Device/FitImage)
+	$(call Device/UbiFit)
+	DEVICE_VENDOR := TP-Link
+	DEVICE_MODEL := Deco X80-5G
+	BLOCKSIZE := 128k
+	PAGESIZE := 2048
+	DEVICE_DTS_CONFIG := config@hk01.c5
+	SOC := ipq8074
+	DEVICE_PACKAGES := kmod-hwmon-gpiofan ipq-wifi-tplink_deco-x80-5g \
+	 	 kmod-usb-serial-option kmod-usb-net-qmi-wwan kmod-usb-serial-qualcomm uqmi\
+		 usbutils kmod-usb-storage
```

Comment:
Why do you need kmod-usb-storage and usbutils by default?

Reply:
ill drop them yip it does not need usb storage at all but the tools one was handy to diag when I was getting the modem initially peeking and poking with the modem but can be removed now.